src/System.IO.Compression/tests/ZipArchive/zip_ManualAndCompatibilityTests.cs

```diff
@@ -45,7 +47,7 @@ public class zip_ManualAndCompatabilityTests : ZipFileTestBase
         /// For example, the file "aa\bb\cc\dd" in a zip created on Unix should be one file "aa\bb\cc\dd" whereas the same file
         /// in a zip created on Windows should be interpreted as the file "dd" underneath three subdirectories.
         /// </summary>
-        [Theory]
+        [ConditionalTheory(nameof(IsUsingNewPathNormalization))]
```

Comment:
I don't see them failing. Only this one, and this is the one associated to the issue.